### DISCUSSION TITLE
Parametric aligned umap?

### DISCUSSION THREAD

**Discussion by IanQS** [lmcinnes/umap#1124] [General]
I'm interested in first training a AlignedUmap on some time-series dataset. I then want to use the trained AlignedUmap to run inference on some new points to see if the new points lie on the surface learned by the Umap. In a standard UMAP, I can just use the ParametricUmap to learn the encoder and decoder, but is this possible with AlignedUmap? I checked the source code and didn't find anything about passing in an encoder or decoder. 

Thank you!

**Comment by lmcinnes** [comment]
AlignedUMAP and ParametricUMAP don't currently fit together, so I don't quite think that will be possible I'm afraid. Sorry.
